File: multilibrary/src/main/java/com/zhongjh/albumcamerarecorder/camera/listener/ErrorListener.java
package com.zhongjh.albumcamerarecorder.camera.listener;

/**
 * 处理异常的Listener
 * @author zhongjh
 */
public interface ErrorListener {

    /**
     * 发现异常
     */
    void onError();

    /**
     * 发现权限异常
     */
    void onAudioPermissionError();

}

File: multilibrary/src/main/java/com/zhongjh/albumcamerarecorder/camera/listener/ErrorListener.kt
package com.zhongjh.albumcamerarecorder.camera.listener

/**
 * 处理异常的Listener
 * @author zhongjh
 */
interface ErrorListener {
    /**
     * 发现异常
     */
    fun onError()

    /**
     * 发现权限异常
     */
    fun onAudioPermissionError()
}